xp/xconnect/startup.ps1

```diff
@@ -0,0 +1,13 @@
+$licenseFile = "C:/license/license.xml"
+if(Test-Path $licenseFile)
+{
+    Copy-Item -Path $licenseFile -Destination "C:/inetpub/wwwroot/xconnect/App_Data"
```

Comment:
Hiervoor zouden we een config file gebruiken toch?

Reply:
Het is makkelijker om dit te kopiëren. Dan kan de config ook hetzelfde blijven.